zebrad/src/components/sync/end_of_support.rs

```diff
@@ -13,7 +13,7 @@ use zebra_chain::{
 use crate::application::release_version;
 
 /// The estimated height that this release will be published.
-pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_626_500;
+pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_678_363;
```

Comment:
When i changed this the tip was `2_677_211`, i am giving 1 day more (+`1152`) = `2_678_363`